libraries/botbuilder/tests/streaming/botFrameworkAdapterStreaming.test.js

```diff
@@ -298,53 +222,48 @@ describe('BotFrameworkAdapter Streaming tests', () => {
             expect(response.statusCode).to.equal(501);
         });
 
-        it('returns a 500 when bot can not run', async () => {
-            const MiddleWare = require('botbuilder-core');
-            const bot = {};
-            let mw = {
```

Comment:
The logic in this test was out of date, as we no longer pass in a MiddlewareSet to the constructor. The 500 was probably being thrown due to `BotFrameworkAdapter.logic` being missing, so we should harden the rewritten test and the [test](https://github.com/microsoft/botbuilder-js/pull/1433/files#r348312596) that follows it.